### DISCUSSION TITLE
Stable vcpkg release?

### DISCUSSION THREAD

**Discussion by aggieNick02** [microsoft/vcpkg#20687] [Q&A] [answered]
I asked this about 8 months ago (https://github.com/microsoft/vcpkg/issues/16365), and am just wondering if the guidance is still the same. If I want to use vcpkg and minimize the chance that I end up with a build with a bug in the actual vcpkg functionality, what should I do? Should I be using the latest release, which was in May? Or is the most current commit just as likely to be stable?

**Comment by autoantwort** [comment, marked as answer]
The guidance is still the same. Don't use the release from May. Always use the latest version from the master branch. Every change is tested in ci and reviewed so the master branch should be stable.

**Reply by aggieNick02** [reply]
Great, thanks for the help!
